.travis.yml

```diff
@@ -1,9 +1,9 @@
 
 if: (branch = master) OR (branch = development) OR (tag IS present)
+dist: bionic
 language: node_js
 node_js: '12'
 before_install:
-  - yarn install --network-concurrency 1
```

Comment:
Not sure about removing this line. If the package.json has 2 dependencies from Github (instead of npm), the installation will fail.

Reply:
Why is it not a problem in safe-react then?

Reply:
I don't think the concurrency is somehow related to where the dependencies are coming from